### PR TITLE
Dissallow empty rosters

### PR DESCRIPTION
**Pull request by nwoodthorpe**
We should show the user an error if they create a roster with no students

### CODE REVIEW

File: app/controllers/rosters_controller.rb [hunk 1]
```diff
@@ -12,16 +12,21 @@ def new
 
   def create
     @roster = Roster.new(identifier_name: params[:identifier_name])
-    @roster.save!
 
     add_identifiers_to_roster
 
-    @organization.roster = @roster
-    @organization.save!
+    if @roster.roster_entries.empty?
```
Comment (tarebyte) [inline app/controllers/rosters_controller.rb]: This feels like something that should be part of the model, or a new creator. WDYT?

Comment (nwoodthorpe) [inline app/controllers/rosters_controller.rb]: Yeah you're right, I can enforce this on the model. :+1:

File: app/controllers/rosters_controller.rb [hunk 2]
```diff
@@ -12,16 +12,21 @@ def new
...
+      @roster.errors.add(:roster_entries, "must include at least one student")
```
Comment (nwoodthorpe) [inline app/controllers/rosters_controller.rb]: Going to ignore this

**Comment by nwoodthorpe** [comment]
@tarebyte updated

**Comment by nwoodthorpe** [comment]
Also made sure we have no rosters with 0 entries on production: 

```
pry(main)> Roster.all.select{|r| r.roster_entries.empty?}
=> []
```

**Pull request merged**